bazel/repositories.bzl

```diff
@@ -112,10 +112,13 @@ def _go_deps(skip_targets):
 def _rust_deps():
     external_http_archive(
         "rules_rust",
-        patches = ["@envoy//bazel:rules_rust.patch", "@envoy//bazel:rules_rust_ppc64le.patch"],
```

Comment:
is the ppc patch no longer needed?

i know in my previous prs i just deleted - but i was kinda intending to add it back